buildSrc/src/main/groovy/Docker.groovy

```diff
@@ -164,6 +164,11 @@ class Docker {
          */
         String platform;
 
+        /**
+         * Optional set of environment variables to set on the container.
+         */
+        Map<String, String> envVars;
```

Comment:
this seems unused/unrelated?

Reply:
It was used for passing ENV vars into Dockerfiles, but I no longer needed it b/c I'm building the demo images in a different repo. 

I can remove it if you guys don't want to leave the plumbing there.

Reply:
Do you want me to remove this or not? If someone ever does want to use it, the wiring is there now.  I don't actually care, just want a decision on stay-or-go.